packages/ui/src/components/Snippet/index.tsx

```diff
@@ -8,16 +8,25 @@ import { Stack } from '../Stack'
 import { Text } from '../Text'
 
 const LINES_BREAK_REGEX = /\r\n|\r|\n/
+const LINE_HEIGHT = 20
```

Comment:
might be better to get that value from the theme instead of setting a global fixed value